test-case/check-reboot.sh

```diff
@@ -0,0 +1,135 @@
+#!/bin/bash
+
+##
+## Case Name: test reboot
+## Preconditions:
+##    N/A
+## Description:
+##    run reboot for the test
+## Case step:
+##    1. check system status is correct
+##    2. wait for the delay time
+##    3. trigger for the reboot
+## Expect result:
+##    Test execute without report error in the LOG
+##
+
+source $(dirname ${BASH_SOURCE[0]})/../case-lib/lib.sh
+
+OPT_OPT_lst['l']='loop'     OPT_DESC_lst['l']='loop count'
+OPT_PARM_lst['l']=1         OPT_VALUE_lst['l']=3
+
+OPT_OPT_lst['d']='delay'    OPT_DESC_lst['d']='delay after system bootup'
+OPT_PARM_lst['d']=1         OPT_VALUE_lst['d']=2
+
+func_opt_parse_option $*
+
+func_lib_check_sudo
+loop_count=${OPT_VALUE_lst['l']}
+delay=${OPT_VALUE_lst['d']}
+
+# write the orig & status count to the status file
+status_log=$LOG_ROOT/status.txt
+echo $loop_count >> $status_log
+count=$(head -n 1 $status_log|awk '{print $1;}')
+# because write record by desc
```

Comment:
not sure what does this comment mean, the code is self-explanatory enough, maybe just delete it?